pkg/notes/model/note.ts

```diff
@@ -87,6 +90,11 @@ export class Note {
     return this.originalNoteID;
   }
 
+  private readonly attachmentFileID: MediumID[];
+  getAttachmentFileID(): MediumID[] {
+    return this.attachmentFileID;
+  }
```

Comment:
```suggestion
  getAttachmentFileID(): readonly MediumID[] {
    return this.attachmentFileID;
  }
```
`readonly` が無いので, このままだとモデルの外部から制約を無視して書き換えできます.
もしこの ID リストがイミュータブルになるよう定義したいのであれば, `attachmentFileID` の定義の型にも `readonly` を付けるべきです.